docs/ga-requirements.md

```diff
@@ -0,0 +1,51 @@
+P1 (e.g. cannot GA without these):
+* End-to-end tests
+  * OTLP, Jaeger and Zipkin
+  * Spring Boot and Wildfly
+    * (Wildfly chosen due to common javaagent issues around jboss modules and jboss logging)
+  * Java 8, 11, and the latest Java version
+* Benchmarking
+  * Runtime overhead benchmark
+  * Startup overhead benchmark
+* Implement all required semantic attributes
```

Comment:
There is this idea: https://github.com/open-telemetry/opentelemetry-specification/issues/653#issuecomment-662156100